designs/2022-docs-information-architecture-update/README.md

```diff
@@ -0,0 +1,358 @@
+
+As a result of these changes, all personas will be able to better navigate the ESLint documentation and better complete the tasks that they're using ESLint for.
+
+<!-- Why are we doing this? What use cases does it support? What is the expected
+outcome? -->
+
+## Detailed Design
+
+N/A. All changes documentation related.
+
+## Documentation
+
+The newly proposed information architecture should consist of the following sections and pages:
+
+### [REFACTOR SECTION]  Use ESLint in Your Project
+
+- [RENAME SECTION] Use ESLint in Your Project
+  - Rename of current section User Guide
+- [NO CHANGE] Use ESLint in Your Project landing page
+  - Current page: [User Guide landing page](https://eslint.org/docs/latest/user-guide/)
+  - Hidden in sidebar
+- [NO CHANGE] Getting Started
+  - Current page: [Getting Started](https://eslint.org/docs/latest/user-guide/getting-started)
+  - NOTE: I think there’s further work that can be done to improve the getting started experience, but do not want to touch the subject as part of the information architecture changes project. Investigate further in **Phase 3: "Use ESLint in Your Project" documentation update** of the documentation update project.
+- [NO CHANGE] Core Concepts
+  - Current page: [Core Concepts](https://eslint.org/docs/latest/user-guide/core-concepts)
+- [RENAME PAGE] Configure
+  - Current page: [Configuring](https://eslint.org/docs/latest/user-guide/configuring/)
+  - [RENAME PAGES] Rename the child pages of Configuring as follows:
+    - [NO CHANGE] Configuration Files (New)
+    - [NO CHANGE] Configuration Files
+    - [RENAME PAGE] Configuring Language Options -> Configure Language Options
+    - [RENAME PAGE] Configuring Rules -> Configure Rules
+    - [RENAME PAGE] Configuring Plugins -> Configure Plugins
+    - [RENAME PAGE] Ignoring Code -> Ignore Code
+  - Do not change the "Configuring" page and its children pages. Nicholas mentioned that there’s some major revamping to how configuration files work in ESLint, so let’s leave this subsection alone for now.
+- [RENAME PAGE] Command Line Interface Reference
+  - Current page: [Command Line Interface](https://eslint.org/docs/latest/user-guide/command-line-interface)
+- [RENAME PAGE] Rules Reference
+  - Current page: [Rules](https://eslint.org/docs/latest/rules/)
+- [RENAME PAGE] Formatters Reference
+  - Current page: [Formatters](https://eslint.org/docs/latest/user-guide/formatters/)
+- [NO CHANGE] Integrations
+  - Current page: [Integrations](https://eslint.org/docs/latest/user-guide/integrations)
+- [RENAME PAGE] Migrate to v8.x
+  - Current page: [Migrating to v8.0.0](https://eslint.org/docs/latest/user-guide/migrating-to-8.0.0)
+
+### [REFACTOR SECTION]  Extend ESLint
+
+- [NO CHANGE] Extend ESLint landing page
+  - Current page:  [Developer Guide](https://eslint.org/docs/latest/developer-guide/)
+  - Hidden in sidebar
```

Comment:
We should move some content from this page to the contributing landing page, as it's currently a mix of both.

Reply:
good point. i can add a note about this.